### PR TITLE
jinja2cpp: bump dependencies + honor msvc runtime + fix CMake imported target

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **jinja2cpp/all**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

closes https://github.com/conan-io/conan-center-index/issues/4479

### CODE REVIEW

**Review by jgsogo — APPROVED** [review]

File: recipes/jinja2cpp/all/conanfile.py [hunk 1]
```diff
@@ -89,7 +95,32 @@ def package(self):
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib", "jinja2cpp"))
         tools.rmdir(os.path.join(self.package_folder, "share"))
+        self._create_cmake_module_alias_targets(
```
Comment (SSE4) [inline recipes/jinja2cpp/all/conanfile.py:98]: is there a PR for that opened already?

**Review by SSE4 — APPROVED** [review]

File: recipes/jinja2cpp/all/conanfile.py (hunk 1 above)
Comment (SpaceIm) [reply recipes/jinja2cpp/all/conanfile.py:98]: I don't think so, you mean in conan?

Comment (SSE4) [reply recipes/jinja2cpp/all/conanfile.py:98]: yes.

**Review by uilianries — APPROVED** [review]

File: recipes/jinja2cpp/all/conanfile.py (hunk 1 above)
Comment (prince-chrismc) [reply recipes/jinja2cpp/all/conanfile.py:98]: there's a PR for this, I found this https://github.com/conan-io/conan/pull/6438

But there's another one Space and Madebr have commented on that I did not find

**Review by prince-chrismc — APPROVED** [review]

File: recipes/jinja2cpp/all/conanfile.py (hunk 1 above)
Comment (SpaceIm) [reply recipes/jinja2cpp/all/conanfile.py:98]: @prince-chrismc not this PR (which has been reverted by the way).

There is an issue opened: https://github.com/conan-io/conan/issues/8360, but no PR AFAIK

Comment (SSE4) [reply recipes/jinja2cpp/all/conanfile.py:98]: okay, I've created one: https://github.com/conan-io/conan/pull/8533

**Pull request merged**